src/concrete/ml/onnx/convert.py

```diff
@@ -149,6 +150,14 @@ def get_equivalent_numpy_forward_from_torch(
         input_names=arguments,
     )
     equivalent_onnx_model = onnx.load_model(str(output_onnx_file_path))
+
+    # Check if the inputs are present in the model's graph
+    for input_name in arguments:
+        assert_true(
+            any(input_name == node.name for node in equivalent_onnx_model.graph.input),
+            f"Input '{input_name}' is not present in the ONNX model. Please check the onnx graph.",
```

Comment:
I would maybe:
- say `Input '{input_name}' (found in the forward pass)` to be more precise 
- detail a bit more `Please check the onnx graph.` : what should the user do ? if he just builds a torch model, saves it to onnx but then encounters this problem, what could likely be the reason for it ?

Reply:
> say Input '{input_name}' (found in the forward pass) to be more precise

```
f"'{input_name}' is missing in the ONNX graph after export. Verify the forward pass for issues."
```

But  about

> detail a bit more Please check the onnx graph. : what should the user do ? if he just builds a torch model, saves it to onnx but then encounters this problem, what could likely be the reason for it ?

I have no idea. For the test we added it was to not use numpy in the forward but I am sure there are tons of different ways to make an onnx input disappear and so as much reasons.

Reply:
ah yes I see, well ok we can keep it that way I guess. anyway yes for the first point I like your suggestion

Reply:
I changed the message. Should be good.